beancount/core/number.py

```diff
@@ -81,27 +87,28 @@ def D(strord=None):
     ignored, as they are assumed to be thousands separators (the French comma
     separator as decimal is not supported). This function just returns the
```

Comment:
The remark "the French comma separator as decimal is not supported" should be removed, now that this new option is introduced.

Reply:
Also, for users from countries where the comma is used as the decimal separator, it would be useful to clarify in the docstring what Beancount's default decimal separator is.